container/container_unix.go

```diff
@@ -399,7 +399,7 @@ func copyExistingContents(source, destination string) error {
 		// destination is not empty, do not copy
 		return nil
 	}
-	return fs.CopyDir(destination, source)
+	return fs.CopyDir(destination, source, fs.WithAllowXAttrErrors())
```

Comment:
I think we should make our own handler here and explicitly check for ENOTSUP or possibly EPERM.
WDYT? @AkihiroSuda ?

Reply:
@cpuguy83 imo the best approach would be to check-before-set, especially for ENOTSUP. In that way we could horizontally control whether we fail for unsupported operations or not. However, I 've sent a change reflecting your comment to have an example to discuss. Intuitively, it doesn't look very nice as I think we 're going too deep, but I 've no other arguments against this since syscall errno is also examined elsewhere.